### PR TITLE
Filtering for graphic, fixes #60, #57, #58

### LINKED ISSUE #60: Allow to (de)select certain ontologies for full display of classes, properties and instances
Currently one can have full listing of classed, properties and individuals in either all or none of the ontologies. Suggest the following options.

- Allow to specify exactly what ontologies to list all concepts for, and none otherwise (e.g. only show pleoMain)
- Allow to specify to specify that all  BUT specific ontologies will have their concepts listed.  E.g .shoe evertying but skos and gist.

### PR DESCRIPTION
**Pull request by sa-bpelakh**
See README changes for what was added.

### CODE REVIEW

**Review by uscholdm — CHANGES_REQUESTED** [review]
A few questions inline.  Suggest adding 2 or 3 examples showing off the major features. Copy and edit much faster than figuring it out from scratch. It would take me a while to figure this out from scratch and get it working, .

File: README.md [hunk 1]
```diff
@@ -135,15 +141,56 @@ optional arguments:
...
+                        option may result in an incomplete version of the
+                        diagram, missing certain links.
+  --predicate-threshold PREDICATE_THRESHOLD
+                        Ignore predicates which occur fewer than
+                        PREDICATE_THRESHOLD times (default 10)
+
+Filters (only one can be used):
+  --include [INCLUDE [INCLUDE ...]]
+                        If specified for --schema, only ontologies matching
+                        the specified URIs will be shown in full detail. If
```
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply README.md:168]: When working with data in input files (as opposed to in a triple store), onto_tool assumes one ontology per file. Once that ontology URI is determined, if it is either not present on the include list or is present on the exclude list, entities from that ontology will not be rendered in the diagram.

When working with data in the triple store, the same filtering logic will be applied to named graphs.

File: README.md [hunk 2]
```diff
@@ -135,15 +141,56 @@ optional arguments:
...
+  --instance-limit INSTANCE_LIMIT
+                        Specify a limit on how many triples to consider that
+                        use any one predicate to find (default 500000). This
+                        option may result in an incomplete version of the
+                        diagram, missing certain links.
+  --predicate-threshold PREDICATE_THRESHOLD
+                        Ignore predicates which occur fewer than
+                        PREDICATE_THRESHOLD times (default 10)
+
+Filters (only one can be used):
```
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply README.md:165]: --schema and --data are mutually exclusive - only one type of graph can be generated at a time.

File: README.md [hunk 3]
```diff
@@ -135,15 +141,56 @@ optional arguments:
...
+  --include [INCLUDE [INCLUDE ...]]
```
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply README.md:166]: It's not nesting, it's a list of includes.

**Pull request merged**